README.md

```diff
@@ -1,15 +1,73 @@
-# GitHub Repo Template
+# Audio ADC - LabVIEW
+
+## Overview
+
+'Audio ADC - LabVIEW' is a MeasurementLink based plugin that enables users to perform measurements of Audio Analog to Digital Converters.
+
+The measurement plugin generate an analog audio signal input to the Device Under Test and measures the Digital Output.
```

Comment:
Rephrase as "This plugin generates analog signal and measures the digital audio output of the ADC".
FYI: We are not generating Audio signal, we are acquiring audio signal.